README.rst

```diff
@@ -30,7 +30,7 @@ email address when accessing the API. As a convenience you can specify this info
 a minid configuration file (`~/.minid/minid-config.cfg`)
 
 
-.. code-block:: none
+.. code-block:: ini
```

Comment:
Small note that `none` causes an error, so I changed it to a valid Pygments entry `ini`. Different Rest Viewers display ini differently though, and it looks like PYPI puts kinda ugly red lines around the values. I could change this to empty to not render Pygments if that would be desirable.

Reply:
Merged, nvm